### PR TITLE
[Version] Bump version to v0.5.1

### PR DESCRIPTION
**Pull request by HaiyiMei**
- Add demo video
- Fix typo
- Fix misc bugs

### CODE REVIEW

File: src/XRFeitoriaUnreal/Content/Python/utils.py [hunk 1]
```diff
@@ -118,8 +118,8 @@ def import_asset(path: Union[str, List[str]], dst_dir_in_engine: Optional[str] =
         asset_tools.import_asset_tasks(import_tasks)
         asset_paths.extend([path.split('.')[0] for path in import_task.get_editor_property('imported_object_paths')])
 
-    unreal.EditorAssetLibrary.save_directory(dst_dir_in_engine, False, True)  # save assets
-    unreal.log(f'Imported assets: {asset_paths}')
+        unreal.EditorAssetLibrary.save_directory(dst_dir, False, True)  # save assets
+        unreal.log(f'Imported asset: {p}')
```
Comment (yl-1993) [inline src/XRFeitoriaUnreal/Content/Python/utils.py:122]: L88, `p` -> `path`, `path` -> `paths` or `path_list`

File: xrfeitoria/actor/actor_base.py [hunk 2]
```diff
@@ -89,6 +89,11 @@ def import_from_file(
         if scale:
             Validator.validate_vector(scale, 3)
 
+        # judge file path
+        file_path = Path(file_path).resolve()
+        if not file_path.exists():
+            raise FileNotFoundError(f'File "{file_path.as_posix()}" not found')
```
Comment (yl-1993) [inline xrfeitoria/actor/actor_base.py:95]: File xxx `is` not found

File: README.md [hunk 3]
```diff
@@ -20,15 +20,13 @@
...
-    <img src="https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrfeitoria/pics/demo.gif"/>
-</div>
+https://github.com/openxrlab/xrfeitoria/assets/35397764/3a4cbc16-de85-48fa-9e15-3b6c0c3b805d
 
 ### Major Features
 
 - Support rendering photorealistic images with ground-truth annotations.
 - Support multiple engine backends, including [Unreal Engine](https://www.unrealengine.com/) and [Blender](https://www.blender.org/).
-- Support assets/camera management, including import, export, and delete.
+- Support assets/camera management, including import, import, export, and delete.
```
[the remark this answers is not in the document]
Comment (HaiyiMei) [reply README.md]: fixed in [`e02c4e4` (#8)](https://github.com/openxrlab/xrfeitoria/pull/8/commits/e02c4e4fa3e5aec00688c56c24b86fe1b9dacbac)

File: src/XRFeitoriaUnreal/Content/Python/utils.py (hunk 1 above)
Comment (HaiyiMei) [reply src/XRFeitoriaUnreal/Content/Python/utils.py]: fixed in [`e02c4e4` (#8)](https://github.com/openxrlab/xrfeitoria/pull/8/commits/e02c4e4fa3e5aec00688c56c24b86fe1b9dacbac)

File: xrfeitoria/actor/actor_base.py (hunk 2 above)
Comment (HaiyiMei) [reply xrfeitoria/actor/actor_base.py]: fixed in [`e02c4e4` (#8)](https://github.com/openxrlab/xrfeitoria/pull/8/commits/e02c4e4fa3e5aec00688c56c24b86fe1b9dacbac)

**Review by yl-1993 — APPROVED** [review]

**Pull request merged**